www-servers/nginx-unit/nginx-unit-1.14.0.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=(python2_7 python3_{5,6,7})
+
+inherit flag-o-matic python-single-r1
+
+MY_P="unit-${PV}"
+DESCRIPTION="Dynamic web and application server"
+HOMEPAGE="https://unit.nginx.org"
+SRC_URI="https://unit.nginx.org/download/${MY_P}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+MY_USE="perl python ruby"
+MY_USE_PHP="php5-6 php7-1 php7-2 php7-3"
+IUSE="${MY_USE} ${MY_USE_PHP} ssl"
+REQUIRED_USE="|| ( ${IUSE} )
+	python? ( ${PYTHON_REQUIRED_USE} )"
+
+DEPEND="perl? ( dev-lang/perl:= )
+	php5-6? ( dev-lang/php:5.6[embed] )
+	php7-1? ( dev-lang/php:7.1[embed] )
+	php7-2? ( dev-lang/php:7.2[embed] )
+	php7-3? ( dev-lang/php:7.3[embed] )
```

Comment:
Additionally, the changelog mentioned PHP 7.4 support. Can you please test and add it accordingly?

Reply:
Unfortunately I don't currently have a test environment with PHP 7.4 available, so this might be something better suited for a -r1 version of the ebuild.

Reply:
Note that adding PYTHON or PHP targets doesn't require revbumping. If you have other incoming fixes in mind though...